cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -1667,6 +1710,18 @@ TSStatus forwardPlan(List<PartitionGroup> partitionGroups, PhysicalPlan plan) {
         status = forwardPlan(plan, partitionGroup);
       }
       if (status.getCode() != TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
+        if (plan instanceof InsertPlan
```

Comment:
It seems that this function `forwardPlan` will only be used in `processNonPartitionedDataPlan`, which will be called only when the type of plan is `DeletePlan` or `DeleteTimeSeriesPlan`, so maybe these new code in this function is useless?

Reply:
You are right. Thanks for your reminding. These codes will be moved to `TSStatus forwardPlan(Map<PhysicalPlan, PartitionGroup> planGroupMap, PhysicalPlan plan)`.